tests/testthat/test-github.R

```diff
@@ -0,0 +1,45 @@
+test_that("use_github_links populates empty URL field", {
+  create_local_package()
+  use_git()
+  local_mocked_bindings(
+    gh_tr = function(tr) {
+        function(endpoint, ...) list(html_url = "https://github.com/USER/REPO")
+      },
+    target_repo = function(github_get) NULL
+  )
+
+  # when no URL field
+  use_github_links()
+  expect_equal(proj_desc()$get_urls(), "https://github.com/USER/REPO")
+})
+
+test_that("use_github_links overwrites field when overwrite = TRUE", {
+  create_local_package()
+  use_git()
+  local_mocked_bindings(
+    gh_tr = function(tr) {
+        function(endpoint, ...) list(html_url = "https://github.com/USER/REPO")
+      },
+    target_repo = function(github_get) NULL
+  )
+  # when an existing url, and overwrite = TRUE, should overwrite with repo URL
+  use_description_field("URL", "https://existing.url", overwrite = TRUE)
+  use_github_links(overwrite = TRUE)
+  expect_equal(proj_desc()$get_urls(), "https://github.com/USER/REPO")
+})
+
+test_that("use_github_links appends to URL field when overwrite = FALSE", {
+  create_local_package()
+  use_git()
+  local_mocked_bindings(
+    gh_tr = function(tr) {
+        function(endpoint, ...) list(html_url = "https://github.com/USER/REPO")
+      },
+    target_repo = function(github_get) NULL
+  )
+  # when an existing field, and overwrite = FALSE, should append
+  use_description_field("URL", "https://existing.url", overwrite = TRUE)
```

Comment:
You could have 2 URLs to start with, then let `use_github_links()` add a 3rd. There's a whole bunch of packages with 3+ URLs (including bcdata and gapminder): the pkgdown site, some URL about the data source or API wrapped, the GitHub URL. So those are good real-world inspiration.

So, combined with leaning into desc, the setup would be something like:

```
d <- proj_desc()
d$set_urls(c("https://bcgov.github.io/bcdata/", "https://catalogue.data.gov.bc.ca/"))
d$write()
```